exampleConfig.js

```diff
@@ -70,6 +70,9 @@ Optional Variables:
   keyNameSanitize:  sanitize all stat names on ingress [default: true]
                     If disabled, it is up to the backends to sanitize keynames
                     as appropriate per their storage requirements.
+  filters:          List of timer metrics that will not be sent. Default will send all metrics.
```

Comment:
So the filters are a set of what not to send. But the Javascript `filter` method takes a positive list rather than a negative list, it looks like. That gets confusing below! Could you make this configuration list positive as well?